postgres/count extension/calculate--0.0.1.c

```diff
@@ -0,0 +1,71 @@
+//C program extension to count number of insert, delete, update, select queries Executed
+
+//including required header files
+#pragma once
+#include <stdio.h>
+//for basic postgres elements
+#include "postgres.h"
+//for using macros
+#include "fmgr.h"
+//for using basic elements and datatypes
+#include "utils/builtins.h"
+
+//for pg
+PG_MODULE_MAGIC;
+
+PG_FUNCTION_INFO_V1(count_insert); 
+PG_FUNCTION_INFO_V1(count_delete); 
+PG_FUNCTION_INFO_V1(count_update); 
+PG_FUNCTION_INFO_V1(count_select); 
+PG_FUNCTION_INFO_V1(count_total);
+PG_FUNCTION_INFO_V1(clear_count); 
+
+//code to be executed during loading and unloading of extension
+void _PG_init(void)
+{
+    //considering for default database
+    char db[10] = 'zoho';
```

Comment:
why hardcoding here ?

Reply:
inorder to declare the default database sir

Reply:
when a question is asked, it means something needs to be fixed.